README.md

```diff
@@ -1,118 +1,153 @@
 # Magento Developer Documentation
 
-Welcome! This site contains the latest Magento developer documentation for ongoing Magento 2.x releases. For additional information, see our [Contribution Guide](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md).
+Welcome! This site contains the latest Magento developer documentation for ongoing Magento 2 releases. For additional information, see our [Contribution Guide](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md).
 
 ## Contributors
 
 Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish that goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Magento documentation.
 
-![](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
+![DevDocs contributors](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
 
-# Building this site
+## Building this site
 
-You can build this site locally in the following ways:
+The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator on top of it.
+The toolkit requires a Ruby environment to be run.
 
-- [Installing the project dependencies locally](#build-using-jekyll) (Mac, Linux)
+There are several options available depending on what type and level of virtualization you want to use:
+
+- [Installing the project dependencies locally](#build-using-jekyll)
 - [Using a Docker container](#build-using-docker) (Mac, Linux)
-- [Using a Vagrant virtual machine](#build-using-vagrant) (Mac, Linux, Windows)
-- [Build DevDocs in Windows](#build-devdocs-in-windows) (Windows 7 & 10)
+- [Using a Vagrant virtual machine](#build-using-vagrant)
```

Comment:
Why did you remove the link for Windows? I tested that section thoroughly, as did the Cloud team. It was a request from both inside and outside users.